client/lib/directly/index.js

```diff
@@ -0,0 +1,109 @@
+/**
+ * @file Interface to the third-party Real Time Messaging (RTM) widget from Directly.
+ *
+ * @see ./README.md for a higher-level overview
+ * @see https://cloudup.com/cySVQ9R_O6S for Directly's configuration guide
+ */
+
+/**
+ * External dependencies
+ */
+import config from 'config';
+
+/**
+ * Internal dependencies
+ */
+import { loadScript } from 'lib/load-script';
+
+const DEFAULT_RTM_WIDGET_OPTIONS = {
+	id: config( 'directly_rtm_widget_id' ),
```

Comment:
I'm uneasy about this now. I haven't ever paid attention to it before but this line means that the module can raise an error at import time and crash Calypso. I would really like to consider isolating throwable regions of code info functions which are called and out of modules which are imported.

here this could mean turning the `id` property into a thunk `() => config( … )`; it could mean turning the default options into a getter `const getDefaultOptions = () => ( { id: config( … ), … } )` or it could mean something else.

I'd at least like to pause and reconsider this pattern